src/Raven.Server/Config/Categories/SecurityConfiguration.cs

```diff
@@ -136,7 +136,7 @@ public class SecurityConfiguration : ConfigurationCategory
 
         [Description("Well known certificate thumbprints that will be trusted by the server as cluster admins.")]
         [DefaultValue(null)]
-        [ConfigurationEntry("Security.WellKnownCertificates.Admin", ConfigurationEntryScope.ServerWideOnly)]
+        [ConfigurationEntry("Security.WellKnownCertificates.Admin", ConfigurationEntryScope.ServerWideOrPerDatabase)]
```

Comment:
Are we using that somewhere in database context now?

Reply:
Zero issue, this is only ever checked on server level.
However, if we define it as an env variable, this mess things up for us when creating databases because we have duplicate def

Reply:
It looks like the problem is somewhere else then. This is clearly not a database item. Maybe revert it and create a separate issue for investigation?